src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,71 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  display: none;
+  visibility: hidden;
+  opacity: 0;
+  width: 0;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 0 $iui-sm;
+  overflow-x: hidden;
+  overflow-y: overlay;
+  resize: horizontal;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-right: 1px solid t(iui-color-background-5);
+  }
+
+  &.iui-side-navigation-submenu-visible {
+    display: block;
+    visibility: visible;
+    min-width: $iui-3xl * 2;
+    width: $iui-3xl * 4;
+    max-width: 50vw;
+    opacity: 1;
+  }
+
+  @include iui-transition-group;
+
+  .iui-submenu-header {
+    height: $iui-baseline * 5;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    .iui-submenu-header-label {
+      @include iui-text(title);
+      display: flex;
+      align-items: center;
+      overflow: hidden;
+
+      button {
```

Comment:
I am not sure about this one. Maybe move to HTML. What do you think @veekeys

Reply:
is it even needed? it is for the button which gonna be child of the `iui-header-buttons`, which already has flex-shrink... 
btw, need to rename those classes as in info panel.

Reply:
No no, it is for the buttons inside label:
![image](https://user-images.githubusercontent.com/36186912/133584301-71cd0f80-dca3-47e7-bbfc-0e193706248d.png)

Reply:
ohh... damn.. and it is shrinking while resizing...?

Reply:
Without that rule yes

Reply:
It can be just simple image in there too, I assume..?Or this is always gonna be there a back button nothing else? Then we will need to hardcode that in react.
Then we could just have a separate class for the back-button or something..?

Reply:
`button` has been replaced with `.iui-button`.

Reply:
@bentleyvk @veekeys If this discussion is resolved, feel free to merge this PR.  @mayank99 has approved the PR but wanted to make sure this discussion was resolved prior to merging.

Reply:
Fine by me.